fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java

```diff
@@ -112,6 +112,10 @@ protected static HttpGet getObjMethod(final String id) {
         return new HttpGet(serverAddress + id);
     }
 
+    protected static HttpHead headObjMethod(final String id) {
+        return new HttpHead(serverAddress + id);
+    }
+
```

Comment:
Did you factor this into the subclasses of `AbstractResourceIT` every time a `HEAD` request occurs?